commands/test.js

```diff
@@ -63,6 +63,7 @@ function test(projects, options, config) {
         project.testRunners.forEach((testRunner, index) => {
           log(`[${projectName}] Running testRunner ${index}...`);
           exitCode = exec(testRunner, project.subDirectory);
+          if (exitCode) throw new Error('testRunner failed');
```

Comment:
Ah, you don't want this in tests.js, because otherwise we don't get to merge the JUnit test suite, which causes the test failures to not show up in CircleCI.

If this was to fix "failing pre-reqs should terminate the install", then this is not the fix, instead https://remix123.slack.com/archives/C02UU9C7A/p1499791335785569?thread_ts=1499725293.076720&cid=C02UU9C7A is the fix (as you've seen).

Probably just remove this commit ("actually fail when tests fail").